### ISSUE TITLE
Feature request: Add invert colors functionality 

### ISSUE THREAD

**Issue by hugoppp** [FirstTimeInForever/intellij-pdf-viewer#10]
Having a PDF in my IDE in dark mode feels like looking directly into the sun. 
Functionality to invert PDF colors would be much appreciated.

**Comment by FirstTimeInForever** [comment]
@hugoppp Do you mean inverting colors of the actual document?

**Comment by hugoppp** [comment]
Yes, just inverting the document colors on the view. Many PDF viewers have this feature, most of them let you set background and foreground/text color.

**Comment by FirstTimeInForever** [comment]
Ok, unfortunately, there is no such feature in pdf.js library I am using for actual rendering. But I'll try come up with something.

**Comment by breandan** [comment]
You could try what [this SO answer](https://stackoverflow.com/a/61814602/1772342) suggested, it worked for me in Firefox:

```javascript:(function(){viewer.style = 'filter: grayscale(1) invert(1) sepia(1) contrast(75%)';})()```

**Comment by FirstTimeInForever** [comment]
Sorry for a long delay - I had to do my stuff at work (.
I implemented this feature a week ago, but had some problems with gradle and new IDEA versions.

@breandan Thank you for your suggestion). I apply `filter: grayscale(85%)` to the page and it seems to be looking okay.
<details>
<summary>Preview of applying invert filter with different values (85%, 90%, 100%)</summary>
<img width="1157" alt="pdfviewer-invert-85" src="https://user-images.githubusercontent.com/7830574/99191877-c76c6900-2780-11eb-8f56-37a572353cb9.png">
<img width="1157" alt="pdfviewer-invert-90" src="https://user-images.githubusercontent.com/7830574/99191881-cc311d00-2780-11eb-94e1-178a82cc9e2e.png">
<img width="1157" alt="pdfviewer-invert-100" src="https://user-images.githubusercontent.com/7830574/99191887-ce937700-2780-11eb-8bc1-3ebec62e0d8f.png">
</details>

I can't promise that this method will work for all use cases (it won't work for documents with pictures), but for plain text documents it should work fine.

I uploaded new version and it will be available in a day or two.

**Comment by FirstTimeInForever** [comment]
New version is up! To use invert colors feature you need to set registry flag `pdf.viewer.enableExperimentalFeatures`. After that, you can use `Invert Document Colors` action to apply `filter: invert(<value>%)` to the pages. You can set `<value>` in plugin settings (`Settings > PDF Viewer > Experimental Features > Colors invert intensity`).
For now it will stay in the experimental section, as I want to explore other ways of inverting document colors.
If you have any suggestions, feel free to post them here.

**Comment by breandan** [comment]
@FirstTimeInForever Nice work, thanks for your responsiveness! I was able to confirm this feature works as described.

**Comment by Endor-8** [comment]
I'd like to suggest also adding `hue-rotate(180deg)` to the filter, so the hue is restored after the inversion, e.g.:
`filter: invert(0.85) hue-rotate(180deg)`
Currently, every color is inverted to its complementary (blue → orange and such), as can be seen in the hyperlinks in the screenshots you [posted](https://github.com/FirstTimeInForever/intellij-pdf-viewer/issues/10#issuecomment-727611417), @FirstTimeInForever.
I use this same filter to invert PDF views in Chrome.

**Comment by xsharh** [comment]
the latest version has not this feture?

**Comment by prd-hai-huynh** [comment]
Do we still have this feature in the latest version? 
thank you very much.

**Comment by middlec000** [comment]
PDF Viewer is great! Thank you for maintaining it!

I'm not able to affect the colors of my pdf other than the Background.
I changed the Background color to 090909 and that updated the PDF Viewer background as expected.
Changing the Foreground and Icons colors had no effect.

What I really want is a nice dark theme for viewing pdf's built by LaTeX as I work on them. 
I typically achieve this by color inversion. I don't see an option for that.

I'm using: 
PyCharm 2022.2.2 (Professional Edition)
Build #PY-222.4167.33, built on September 15, 2022

<img width="1094" alt="Screen Shot 2022-09-16 at 6 26 19 PM" src="https://user-images.githubusercontent.com/46512256/190835220-9611ac50-f613-4480-a71d-e065a4521d59.png">
